ember/app/services/emeis-options.js

```diff
@@ -0,0 +1,3 @@
+import Service from "@ember/service";
+
+export default class EmeisOptionsService extends Service {}
```

Comment:
Why?

Reply:
Is needed for the new emeis version.

Reply:
Ah I see, emeis doesn't ship a default service..